students/unsan-Jeong/users/views.py

```diff
@@ -0,0 +1,41 @@
+import json ,re
+
+from django.http     import JsonResponse, HttpResponse
+from django.views    import View
+
+#from django.core.exceptions import ValidationError
+
+from users.models import Users
+
+class SignUp(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        password_length = 8
```

Comment:
파이썬에서 상수는 대문자로만 이루어져 있습니다.
```suggestion
        PASSWORD_LENGTH = 8
```